docs/docs/howto/immutable-deployments.md

```diff
@@ -0,0 +1,68 @@
+Deleting these fields before calling [`contrast set`](./workload-deployment/set-manifest.md) for the first time will render the deployment immutable and/or make Coordinator recovery impossible.
+
+### Preventing manifest updates
+
+To prevent manifest updates, delete the `WorkloadOwnerKeyDigest`.
+Now [set the manifest](./workload-deployment/set-manifest.md) through `contrast set`.
+Once completed, the Contrast Coordinator will no longer accept the workload owner key for future manifest changes.
+The seedshare owner can still force manifest changes through the [recovery mechanism](./workload-deployment/recover-coordinator.md), if they're able to access to the Coordinator's `ConfigMap`s.
+
+Alternatively, specify the `--disable-updates` flag in the `generate` step for the same result.
+
+```sh
+contrast generate --reference-values <platform> --disable-updates resources/
+```
+
+:::warning
+
+This action is irreversible.
+You won't be able to make changes to your deployment after running `contrast set` with an empty `WorkloadOwnerKeyDigest`.
+
+:::
+
+### Preventing decryption of state and recovery
+
+Deleting the `SeedshareOwnerPubKeys` field from the manifest before calling `contrast set` for the first time will make it impossible to recover both the Contrast Coordinator and any state associated with a deployment.
+Normally, the Coordinator [can be recovered after a restart](./workload-deployment/recover-coordinator.md).
+With an empty list of `SeedshareOwnerPubKeys`, this becomes impossible because no keys will be accepted.
+This also preempts [workload secret recovery](../architecture/secrets.md#workload-secrets), and thus makes it impossible to re-open [encrypted persistent storage](./encrypted-storage.md) after a Coordinator restart.
+
+:::warning
+
+This action is irreversible.
+You won't be able to recover the Contrast Coordinator after a restart.
+Further, any associated state will also be irrecoverable.
+Removing the `SeedshareOwnerPubKeys` effectively turns a (persistent) deployment into an ephemeral one.
+
+**It's highly unlikely that you want to use this feature in conjunction with the encrypted persistent storage feature.**
+
+:::
+
+### Verifying immutability
+
+[Deployment verification](./workload-deployment/deployment-verification.md) works as usual.
+The `contrast verify` step for a manifest with removed `WorkloadOwnerKeyDigest` and/or `SeedshareOwnerPubKeys` will only succeed if the manifest set on the Coordinator is also missing the corresponding keys,
+thereby guaranteeing no party can update and/or recover the deployment and its state.
+
+Additionally, deletion of the `WorkloadOwnerKeyDigest` and `SeedshareOwnerPubKeys` needs to occur before the first use of `contrast set`.
+Setting the manifest and verifying the manifest history will both fail if these keys have been changed.
```

Comment:
* Only set will fail, verify is not relevant for this detail.
* Workload owner keys can be changed after the first `set`, but we should still recommend to start with none if immutability is desired.